### DISCUSSION TITLE
Panda Studio build breaks with pnpm

### DISCUSSION THREAD

**Discussion by rajat2502** [chakra-ui/panda#2757] [General]
### Description

Whenever I try to run the studio build command with pnpm it breaks saying that it cannot find "x" package. When I digged deeper these were the packages that the studio was using as its dependencies. For some reason, pnpm is not able to install them. 
As soon as I switched to yarn the build command worked perfectly - this also reflected in the `node_modules` as it had all the dependencies needed for the build.
<img width="1140" alt="Screenshot 2024-07-26 at 12 36 12 PM" src="https://github.com/user-attachments/assets/0b3a5783-c171-43ad-aaa8-064d23f237a8">

### Link to Reproduction

https://github.com/rajat2502/pandacss-experiments/tree/main/design-system

### Steps to reproduce

1. Go to the folder in the mentioned repo. 
2. Run `pnpm studio:build` command.
3. See error

### JS Framework

React (TS)

### Panda CSS Version

0.37.2

### Browser

CLI

### Operating System

- [X] macOS
- [ ] Windows
- [ ] Linux

**Comment by avegatolber** [comment]
Same here, any updates?

**Comment by literalpie** [comment]
pnpm is more strict about not letting packages access implicit dependencies unless they are declared as dependencies. Try adding kleur as a dependency, using [public-hoist-pattern](https://pnpm.io/npmrc#public-hoist-pattern), or [shamefully-hoist](https://pnpm.io/npmrc#shamefully-hoist).
